css/component_styles/image.css

```diff
@@ -0,0 +1,11 @@
+/* ------- image css ------ */
+
+.img-responsive {
```

Comment:
These properties can be applied across all `img`s using `reset.css`
Not mandatory but an idea

Reply:
ok sure !